exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -215,4 +237,8 @@ private HashCode submitTransaction(Transaction tx) {
   private void checkBlockchainInitialized() {
     checkState(node != null, "Service has not been fully initialized yet");
   }
+
+  private CreateCounterTx createCounter(String name, Fork fork) {
```

Comment:
I'd keep the private methods in some logical order too — usually, in the call order (after the callee).